subworkflows/local/igv_tracks.nf

```diff
@@ -0,0 +1,38 @@
+//
+// IGV TRACKS
+//
+
+include { JUNCTION_TRACK   } from '../../modules/local/junction_track'
+include { UCSC_WIGTOBIGWIG } from '../../modules/nf-core/ucsc/wigtobigwig/main'                                                                          
+
+workflow IGV_TRACKS {
+    take:
+        wig
+        path_sizes
+        spl_junc
+
+    main:
+        ch_versions = Channel.empty()
+
+        ch_wig = wig
+            .map{ meta, wigs ->
+            return[meta, wigs[1]] }
+
+        UCSC_WIGTOBIGWIG(
+            ch_wig,
+            path_sizes
+        )
+        ch_versions = ch_versions.mix(UCSC_WIGTOBIGWIG.out.versions.first())
+
+        JUNCTION_TRACK(
+            spl_junc
+        )
```

Comment:
The output here should be bgzipped and indexed. Next PR?

Reply:
Yes, next PR, I didn't want to add another step XD

Reply:
In the end I decided to add it here too, as it was more straightforward than I thought at first :)